lib/storage/clusterconfig/clusterconfig.go

```diff
@@ -430,7 +446,23 @@ const specSchemaTemplate = `{
             },
             "extraArgs": {"type": "array", "items": {"type": "string"}}
           }
-        }
+		},
+		"gravityControllerService": {
```

Comment:
nit: this looks a bit off indentation-wise. I believe the rest of the YAML uses `ts=2 sts=2 sw=2 expandtab` (in vim terms) and this is using tabs instead of whitespace.